src/pages/en/guides/markdown-content.mdx

```diff
@@ -449,14 +441,8 @@ import { rehypeAccessibleEmojis } from 'rehype-accessible-emojis';
 export default {
   markdown: {
     // Applied to .md and .mdx files
-    remarkPlugins: [remarkToc],
-    // Preserves remark-gfm and remark-smartypants
-    extendDefaultPlugins: true,
+    remarkPlugins: [remarkToc, rehypeAccessibleEmojis],
   },
-  integrations: [mdx({
-    // Applied to .mdx files only
-    rehypePlugins: [rehypeAccessibleEmojis],
```

Comment:
Note: if this were `remarkPlugins`, MDX would blow away existing Markdown plugins and require the user to reapply them. Removing this MDX extension example to leave explanation to a separate section.